website/docs/guides/markdown-features/markdown-features-math-equations.mdx

```diff
@@ -128,6 +128,19 @@ module.exports = {
 };
 ```
 
+## Self-hosting KaTeX assets
+
+Loading stylesheets, fonts, and javascript libraries from CDN sources is a good practice for popular libraries and assets. Provided such assets are widely used across various websites, those assets will not be downloaded every time we visit a new website that uses the same assets, rather they would be cached by the browser reducing network load. In case you prefer to self-host the `katex.min.css` (along with required KaTeX fonts), you can download the latest version from [KaTeX GitHub releases](https://github.com/KaTeX/KaTeX/releases). Extract and copy the `katex.css` or `katex.min.css` and the `fonts` directory to docusaurus `static` directory. And in `docusaurus.config.js`, replace the css `href` from CDN url to your local path (say, `/katex/katex.css`).
+
+```js
+stylesheets: [
+    {
+      href: '/katex/katex.css',
+      type: 'text/css',
+    },
+  ],
+```
```

Comment:
Let's try to keep the syntax valid, shall we?

```suggestion
```js title="docusaurus.config.js"
module.exports = {
  stylesheets: [
    {
      href: '/katex/katex.css',
      type: 'text/css',
    },
  ],
};
```
```

This way, we get syntax highlighting and precommit formatting.

Reply:
Sure, looks better 👍

Reply:
@Josh-Cena The style lint complains about copyright header and missing generic font-family in `katex.min.css`. If I add copyright header, it de-minifies the css file 🤔